src/constants/abacus.ts

```diff
@@ -293,8 +293,8 @@ export const ORDER_STATUS_STRINGS: Record<KotlinIrEnumValues<typeof AbacusOrderS
   [AbacusOrderStatus.pending.name]: STRING_KEYS.PENDING,
   [AbacusOrderStatus.pending.rawValue]: STRING_KEYS.PENDING,
 
-  [AbacusOrderStatus.untriggered.name]: STRING_KEYS.UNTRIGGERED,
-  [AbacusOrderStatus.untriggered.rawValue]: STRING_KEYS.UNTRIGGERED,
+  [AbacusOrderStatus.untriggered.name]: STRING_KEYS.CREATED,
```

Comment:
oo can we double check with product that it is the case for the status in orders table too? i was under the impression that in the notification it will be shown as "created", but we'll keep it "untriggered" in the table

Reply:
Hmm so these strings are only actually used in notifications! Status in the orders table is just represented by the symbols (no strings)
<img width="161" alt="Screenshot 2024-05-06 at 1 39 39 PM" src="https://github.com/dydxprotocol/v4-web/assets/70078372/c75cb199-268b-4fa3-9075-b5659e19bd9b">

Reply:
o right!! i have short term memory loss!!